addons/website_blog/controllers/main.py

```diff
@@ -113,7 +113,7 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
         # if blog, we show blog title, if use_cover and not fullwidth_cover we need pager + latest always
         offset = (page - 1) * self._blog_post_per_page
         if not blog:
-            if use_cover and not fullwidth_cover:
+            if use_cover and not fullwidth_cover and not tags and not date_begin and not date_end:
```

Comment:
I was wondering why `search` was not here too actually. Found out it is because the search is used on the top page blog post... which leads to this strange behavior:

![image](https://user-images.githubusercontent.com/10338094/200253767-cb55b305-c944-454a-a1ef-e67de318b149.png)

Clearly, we should fix that too doing exactly like `date_begin`/`date_end`: not showing a top post when searching. To see if possible in stable, if not in master. Can you create a task for this, assign yourself and do that in another PR later (you have other urgent OPW that await, let's finish this forward-port, then your other OPWs, then this search stuff)?

(this commit here can thus stay like)